Source/Wpf/Prism.Autofac.Wpf/Prism.Autofac.Wpf.csproj

```diff
@@ -11,6 +11,18 @@
     <AssemblyName>Prism.Autofac.Wpf</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
+    <PrismVersion>7.0.0</PrismVersion>
```

Comment:
Can this be a parameter? Maybe write a "how to release" page and add it to the wiki/repo for things to not to forget? Like bumping this number.

Reply:
yes it could... but if we want to version each project individually then it needs to be defined in the project